draft-ietf-oauth-transaction-tokens.md

```diff
@@ -389,7 +389,7 @@ The figure below {{figleaftxtokenbody}} shows a non-normative example of the JWT
 # Txn-Token Service
 A Txn-Token Service defines a profile of the OAuth 2.0 Token Exchange {{RFC8693}} endpoint that can respond to Txn-Token issuance requests. This profile of the OAuth 2.0 Token Exchange {{RFC8693}} specification MUST be used to obtain Txn-Tokens. The unique properties of the Txn-Token requests and responses are described below. The Txn-Token Service MAY optionally support other OAuth 2.0 endpoints and features, but that is not a requirement for it to be a Txn-Token Service.
 
-Each Trust Domain MUST have exactly one logical Txn-Token Service.
+Each Trust Domain MAY have exactly one logical Txn-Token Service.
```

Comment:
I'm not very familiar with the distinction here, but is there an optionality implied in the word "MAY"? I guess the intent is to say "one and only one logical Txn-Token Service"

Reply:
Perhaps the language should read, "...MUST have at most one logical Txn-Token session"

Reply:
Should we say "Each Trust Domain that uses Txn-Tokens MUST have exactly one logical Txn-Token Service."?

Reply:
That language works for me, @tulshi.